.github/workflows/unit-tests.yaml

```
@@ -59,6 +59,13 @@ jobs:
       with:
         paths: 'junit-reports/report.xml'
 
+    - name: Report junit failures in jira
```

The fact we use JUnit format is an implementation detail and I'm afraid might be confusing more than helpful.

```suggestion
    - name: Report test failures to Jira
```
Same below.